systemd/datadog_checks/systemd/systemd.py

```diff
@@ -0,0 +1,175 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import copy
+
+from collections import defaultdict
+from six import iteritems
+
+import pystemd
+from pystemd.systemd1 import Unit
+
+from datadog_checks.base import AgentCheck, ConfigurationError
+from datadog_checks.base import ensure_unicode
+
+
+class SystemdCheck(AgentCheck):
+
+    UNIT_STATUS_SC = 'systemd.unit.active'
+
+    def __init__(self, name, init_config, instances):
+        if instances is not None and len(instances) > 1:
+            raise ConfigurationError('Systemd check only supports one configured instance.')
+        super(SystemdCheck, self).__init__(name, init_config, instances)
+
+        instance = instances[0]
```

Comment:
Since we can only do A6 here, we can instead do `self.instance.get('x')` where it's called below.

Reply:
And test the ConfigurationError exception in the `__init__` method.